### ISSUE TITLE
F2 doesn't work to rename a file anymore

### ISSUE THREAD

**Issue by IsaacLoop** [microsoft/vscode#177113]
Type: <b>Bug</b>

I select a file, I press F2. And nothing happens. The shortcut is still set (renameFile), with all the default things following.

VS Code version: Code 1.76.1 (5e805b79fcb6ba4c2d23712967df89a089da575b, 2023-03-08T16:32:00.131Z)
OS version: Windows_NT x64 10.0.23403
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.77GB (5.12GB free)|
|Process Argv|--crash-reporter-id 5c4a1586-ff02-4b02-9547-d1bf2c3d5be8|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (59)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-css-formatter|aes|1.0.2
armview|ben|0.4.6
vscode-intelephense-client|bme|1.9.5
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.49.0
githistory|don|0.6.20
gitlens|eam|13.3.2
copilot|Git|1.77.9225
copilot-labs|Git|0.12.791
go|gol|0.38.0
todo-tree|Gru|0.0.224
latex-workshop|Jam|9.7.0
better-cpp-syntax|jef|1.17.2
rainbow-csv|mec|3.5.0
azure-pipelines|ms-|1.208.0
azure-dev|ms-|0.4.0
vscode-azureappservice|ms-|0.24.7
vscode-azurefunctions|ms-|1.10.4
vscode-azureresourcegroups|ms-|0.6.1
vscode-azurestaticwebapps|ms-|0.11.4
vscode-azurestorage|ms-|0.15.1
vscode-azurevirtualmachines|ms-|0.6.4
vscode-bicep|ms-|0.15.31
vscode-cosmosdb|ms-|0.19.3
vscode-docker|ms-|1.24.0
vscode-dotnet-runtime|ms-|1.6.0
isort|ms-|2022.8.0
python|ms-|2023.4.1
jupyter|ms-|2023.2.1200692131
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.282.0
remote-ssh|ms-|0.98.0
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.76.1
azure-account|ms-|0.11.3
azurecli|ms-|0.5.0
cmake-tools|ms-|1.13.45
cpptools|ms-|1.14.4
cpptools-extension-pack|ms-|1.3.0
remote-explorer|ms-|0.2.0
vscode-node-azure-pack|ms-|1.1.1
azurerm-vscode-tools|msa|0.15.8
vscode-paste-image|mus|1.0.4
autodocstring|njp|0.6.1
fix-json|oli|0.1.2
java|red|1.15.0
html-preview-vscode|tht|0.2.5
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.49.0
vscode-java-dependency|vsc|0.21.1
vscode-java-pack|vsc|0.25.9
vscode-java-test|vsc|0.38.1
vscode-maven|vsc|0.40.4
markdown-pdf|yza|1.4.4

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewvextcfv2:30681851
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30662994
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
a9j8j154:30646983
showlangstatbar:30672706
pythonb192:30669360
funwalk2cf:30682975
pythonms35cf:30680985

```

</details>

**Comment by heysujal** [comment]
@IsaacLoop try turning off the extensions and see if it works.

**Comment by IsaacLoop** [comment]
Hi @heysujal, it still doesn't work.

By the way, I just realized the issue is the same for any keyboard shortcut in the VS Code "Folders" tab. `Delete`, `Ctrl+C`, `Ctrl+V.`.. Even though these shortcuts still appear when I right click on a file.

**Comment by yunotruong-novobi** [comment]
same as me
![image](https://user-images.githubusercontent.com/86941956/226295144-7bb87e0a-f3b7-4692-bc02-cad8bb372594.png)

**Comment by shengtian-liao** [comment]
me too.

**Comment by thecodrr** [comment]
This is occurring on my end as well. The problem seems to be that the `Folders` view is not getting activated.

Pressing `Ctrl+C` when a file is selected gives the following output:

```
2023-03-27 22:13:06.709 [info] [KeybindingService]: | Resolving ctrl+[KeyC]
2023-03-27 22:13:06.709 [info] [KeybindingService]: \ From 18 keybinding entries, matched editor.action.clipboardCopyAction, when: no when condition, source: built-in.
2023-03-27 22:13:06.709 [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: KeyC, keyCode: 67, key: c
2023-03-27 22:13:06.710 [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: KeyC, keyCode: 33 ('C')
2023-03-27 22:13:06.710 [info] [KeybindingService]: | Resolving ctrl+[KeyC]
2023-03-27 22:13:06.710 [info] [KeybindingService]: \ From 18 keybinding entries, matched editor.action.clipboardCopyAction, when: no when condition, source: built-in.
2023-03-27 22:13:06.711 [info] [KeybindingService]: + Invoking command editor.action.clipboardCopyAction.
2023-03-27 22:13:06.881 [info] [KeybindingService]: + Ignoring single modifier ctrl due to it being pressed together with other keys
```

`editor.action.clipboardCopyAction` is the wrong action in this case. It should be `filesExplorer.copy`. Seems like a priority issue because things like `Ctrl + Alt + C` do work.

**Comment by vasiloeth** [comment]
Happens the same for me,
Can't rename, copy or paste files inside the Folders views

**Comment by yunotruong-novobi** [comment]
- I removed vscode and settings data from my laptop (follow this introduce [uninstall vscode](https://code.visualstudio.com/docs/setup/uninstall)
- After you reinstall Vscode, do not sync all configurations from your account, you should sync Settings and Extensions to save time.
![image](https://user-images.githubusercontent.com/86941956/228119247-c163982b-960d-4c01-accb-deaaef10b658.png)
- After the data is synced successfully, now you can remove the configurations of your account cloud and turn it on again with all options to sync
![image](https://user-images.githubusercontent.com/86941956/228119502-8bc76785-aaad-4e93-b294-c921ff95425e.png)

**Comment by lramos15** [comment]
This should be fixed in 1.77 please update
